mapping.csv

```diff
@@ -143,14 +143,13 @@
 "^monerominer$",monerominer,
 "^morto$",morto,
 "^multiplug$",multiplug,adware
-"^murofet$",murofet,
 "^murofet$",murofet,malpedia
-"^netsky$",netsky,
+"^netsky$",netsky,https://en.wikipedia.org/wiki/Netsky_(computer_worm)
 "^(neutrino|kasidet)$",neutrino,
 "^njworm$",njworm,
 "^oimobi$",oimobi,on android
 "^padcrypt$",padcrypt,
-"^palevo$",mariposa,
+"^palevo$",palevo,mariposa,butterfly,pilleuz,rimecud,https://www.trendmicro.com/vinfo/us/threat-encyclopedia/malware/palevo
```

Comment:
How are the other mentioned malware names related to palevo?

Reply:
Aliases used by different AV. For Rimecud/Pilleuz see e.g. the TrendMicro writeup. 'Mariposa' is Spanish for 'butterfly'. Both names have been used for the infamous botnet build up with Palevo.